### PR TITLE
i18n(pt-BR): Add `cli-reference.md` translation

### PR DESCRIPTION
**Pull request by Yan-Thomas**
#### What kind of changes does this PR include?

- [ ] Minor content fixes (broken links, typos, etc.)
- [ ] New or updated content
- [x] Translated content
- [ ] Changes to the docs site code
- [ ] Something else!

#### Description

Adds the pt-BR `cli-reference.md` page

### CODE REVIEW

File: src/pages/pt-BR/reference/cli-reference.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+Mostra o número de versão do Astro.
+
+### `--drafts`
+
+Inclui páginas Markdown `draft` na build.
+
+### `--help`
+
+Mostra a mensagem de ajuda.
```
Comment (victor0x16) [inline src/pages/pt-BR/reference/cli-reference.md:121]:
```suggestion
Mostra a mensagem de ajuda e sai.
```

**Review by victor0x16 — CHANGES_REQUESTED** [review]

**Comment by victor0x16** [comment]
@Yan-Thomas , well done 🚀 !

**Comment by Yan-Thomas** [comment]
Thanks @victor0x16!!

**Review by victor0x16 — APPROVED** [review]
LGTM

**Comment by sarah11918** [comment]
Can you just resolve the conflicts with nav.ts again, now that new pages are getting merged  :)

**Comment by Yan-Thomas** [comment]
Solved, @sarah11918!

**Comment by Yan-Thomas** [comment]
Ok, not solved.

**Pull request merged**